ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureBlobDataAccessor.java

```diff
@@ -90,8 +90,12 @@ public AzureBlobDataAccessor(CloudConfig cloudConfig, AzureCloudConfig azureClou
     uploadTimeout = Duration.ofMillis(cloudConfig.cloudUploadRequestTimeout);
     batchTimeout = Duration.ofMillis(cloudConfig.cloudBatchRequestTimeout);
 
-    storageClient = Utils.getObj(azureCloudConfig.azureStorageClientClass, cloudConfig, azureCloudConfig, azureMetrics,
-        blobLayoutStrategy);
+    if (azureCloudConfig.azureStorageAccountInfo != null && !azureCloudConfig.azureStorageAccountInfo.isEmpty()) {
```

Comment:
nit: We could use !Utils.isNullOrEmpty(azureCloudConfig.azureStorageAccountInfo) too.

Reply:
azureStorageAccountInfo is a List type